src/components/InputImage/InputImage.js

```diff
@@ -19,16 +19,22 @@ function InputImage(props) {
   }
 
   return (
-    <label className="InputImage">
+    <div className="InputImage">
+      <label
+        htmlFor="imageInput"
+        className={"InputImage__label " + (image ? 'uploaded' : '')}>
```

Comment:
We use a package named ‘classnames’ to help with dynamic classNames, also by convention state classes should be prefix with ‘is/has’, so in this case ‘is-uploaded’

Reply:
https://www.npmjs.com/package/classnames